autoreduce_frontend/reduction_viewer/views/common.py

```diff
@@ -94,56 +113,83 @@ def prepare_arguments_for_render(arguments: ReductionArguments, instrument: str)
     return final_standard, final_advanced, variable_help
 
 
+def fetch_api_urls(vars_kwargs):
+    """Convert file URLs in vars_kwargs into API URL strings."""
+    for category, headings in vars_kwargs.items():
+        for heading, heading_value in headings.items():
+            if "file" in heading.lower() and isinstance(heading_value, dict):
```

Comment:
What does `"file" in heading.lower()` intend? Does it look for `file` in the variable name? Because we can't guarantee that since users will just name it whatever makes sense